src/htm/algorithms/Connections.hpp

```diff
@@ -818,15 +813,6 @@ class Connections : public Serializable
   void unsubscribe(UInt32 token);
 
 protected:
-  /**
-   * Check whether this segment still exists on its cell.
-   *
-   * @param Segment
-   *
-   * @retval True if it's still in its cell's segment list.
-   */
-  bool segmentExists_(const Segment segment) const;
```

Comment:
The user is responsible for giving the Connections class valid Segments.
Either they keep track of which segments have been created / destroyed,
Or they should query the Connections object to get which segments exist.